### PR TITLE
code cleanup

### PR DESCRIPTION
**Pull request by jurko-gospodnetic**
Generic code cleanup extracted so it does not pollute other pull requests.

### CODE REVIEW

**Comment by gaborbernat** [comment]
Looks good 👌

File: tox/_pytestplugin.py [hunk 1]
```diff
@@ -292,6 +292,24 @@ def fnmatch_lines(self, lines2):
 @pytest.fixture
 def initproj(request, tmpdir):
     """ create a factory function for creating example projects. """
+    # Constructed folder/file hierarchy examples:
```
Comment (gaborbernat) [inline tox/_pytestplugin.py]: Let's make this part of the docstring, otherwise 🆗

Comment (jurko-gospodnetic) [reply tox/_pytestplugin.py]: done

**Comment by jurko-gospodnetic** [comment]
@gaborbernat - change done

**Pull request merged**